### ISSUE TITLE
Inconsistent F1 of ERFNet on CULane-test (much lower than ERFNet-CULane-PyTorch)

### ISSUE THREAD

**Issue by Yibin122** [cardwing/Codes-for-IntRA-KD#2]
Thanks for sharing your latest algorithm!

But I am confused about table 3 in your paper. The F1 measure of ERFNet is only 70.2, which is much lower than 73.1 as previously reported here:
https://github.com/cardwing/Codes-for-Lane-Detection

Why is there such an inconsistency?

**Comment by cardwing** [comment]
The ERFNet model provided in that GitHub repo is trained with several bells and whistles (e.g., warmup pretraining, finetuning, etc). In our CVPR paper, we use the Torch-based ERFNet and it has much lower performance compared with the PyTorch version.

**Comment by Yibin122** [comment]
Then did you try guiding the PyTorch version with a better teacher network other than ResNet-101 (72.8)?

**Comment by cardwing** [comment]
To my knowledge, the PyTorch-based ERFNet model achieves the best F1-measure on CULane-test. I have tried to find a better teacher but failed. You can have a try.
